xclim/testing/tests/test_indices.py

```diff
@@ -1547,67 +1522,59 @@ def test_Celsius(self, tas_series):
         ] += -15
         a[(a.time.dt.season == "MAM") & (a.time.dt.year == 1972)] += -10
 
-        out = xci.tg_mean_warmcold_quarter(a, op="warmest", src_timestep="D")
+        out = xci.tg_mean_warmcold_quarter(a, op="warmest")
         np.testing.assert_array_almost_equal(out, [21.51648352, 25])
 
-        out = xci.tg_mean_warmcold_quarter(a, op="coldest", src_timestep="D")
+        out = xci.tg_mean_warmcold_quarter(a, op="coldest")
         np.testing.assert_array_almost_equal(out, [-14.835165, -9.89011])
 
 
 class TestPrcptot:
     @staticmethod
     def get_data(pr_series):
-        pr = pr_series(np.zeros(365 * 2), start="1971-01-01")
-        pr += 1 / 3600 / 24
-        pr[0:7] += 10 / 3600 / 24
-        pr[-7:] += 11 / 3600 / 24
+        pr = pr_series(np.ones(731), start="1971-01-01", units="mm / d")
+        pr[0:7] += 10
+        pr[-7:] += 11
         return pr
 
     @pytest.mark.parametrize(
-        "freq,units,expected",
+        "freq,expected",
         [
-            (("D", "D"), "mm/day", [435.0, 442.0]),
-            (("7D", "W"), "mm/week", [441.0, 436.0]),
-            (("MS", "M"), "mm/month", [435.0, 442.0]),
+            ("D", [435.0, 443.0]),
+            ("7D", [441.0, 485.0]),
+            ("MS", [435.0, 443.0]),
```

Comment:
Why are the values different ?

Reply:
Because I added a day to the timeseries. See above, before it was 2x 365 days, but 1972 is a leap year, so the timeseries would end on Dec. 30, which I didn't like. I made it 731 days and voilà.